docs/atomic-cluster.1.md

```diff
@@ -0,0 +1,52 @@
+% ATOMIC(1) Atomic Man Pages
+% Matthew Barnes
+% May 2016
+# NAME
+atomic-cluster - Manage cluster hosts
+
+# SYNOPSIS
+**atomic cluster [OPTIONS] SUBCOMMAND**
+
+**atomic cluster** will issue commands to a Commissaire server over HTTP.
```

Comment:
Maybe a small line here to say that it will only be available if Commissaire is installed?